expected/15.0/sqlite_fdw.out

```diff
@@ -17,6 +17,8 @@ CREATE FOREIGN TABLE numbers(a int OPTIONS (key 'true'), b varchar(255)) SERVER
 CREATE FOREIGN TABLE multiprimary(a int, b int OPTIONS (key 'true'), c int OPTIONS(key 'true')) SERVER sqlite_svr;
 --Testcase 136:
 CREATE FOREIGN TABLE noprimary(a int, b text) SERVER sqlite_svr;
+-- readonly/readwrite test github pull 59
```

Comment:
readonly/readwrite is not in this PR anymore. Please update this test.
This PR is only for updatable option.

Reply:
Yes, the main idea of this PR was changed since creating this comment. Fixed.

Reply:
There are some other places has the same comment "readonly/readwrite test github pull 59". Could you fix them all?

Reply:
Please check. Look like there are no more this places.

Reply:
Thanks. Confirmed.